src/features/home/hooks/useUserMutations.tsx

```diff
@@ -0,0 +1,12 @@
+import { useApiClient } from 'core/hooks';
+import { ZetkinLanguage } from '../components/SettingsList';
+
+export default function useUserMutations() {
+  const apiClient = useApiClient();
+
+  const changeUserLanguage = async (lang: ZetkinLanguage) => {
+    await apiClient.patch(`/api/users/me`, { lang });
+  };
```

Comment:
Just want to make double sure that it's intentional to not do anything on the store here?

Reply:
I think it's fine for now, because we refresh the browser after this. But we will probably need to build store logic for user mutations soon.